src/app.js

```diff
@@ -1 +1,43 @@
 'use strict';
+
+const terminal = require('./modules/terminal');
+const fs = require('fs');
+const path = require('path');
+
+function copyFile() {
+  terminal.question('Enter your command: ', (key) => {
+    const [command, firstFile, secondFile] = key.trim().split(' ');
+    const firstFilePath = path.join(__dirname, firstFile);
+    const secondFilePath = path.join(__dirname, secondFile);
+
+    if (command === 'cp'
+      && fs.existsSync(firstFilePath)
+      && !fs.existsSync(secondFilePath)
+    ) {
+      const content = fs.readFileSync(
```

Comment:
This way you are loading the **entire** file into memory. Program will eat a lot of memory and take a long time.

You can test this copying files that are several GBs in size.

Try using streams to fix this

Reply:
we have not learnt streams yet

Reply:
Working with things you haven't learnt yet is a part of the job ;)